src/js/router.js

```diff
@@ -1,18 +1,27 @@
-var env = require('./env'),
-    resources = require('./resources');
+var resources = require('./resources');
+
+var root = require('./env').root;
 
 
 var reverse = function() {
```

Comment:
Can you write a comment about what can be accepted in this function? One of the advantages to the [original implementation](https://github.com/PEDSnet/pedsnet-catalogue/blob/29c82ae342b7b6b4ffdfdde25c8c2145948e2ae7/src/js/router.js) was that it was self-describing. Obviously accounting for all possible URLs is not feasible nor flexible, but use regexes to describe the structure would be a good start. Also, it would be simpler to evaluate and construct that path as before and at the end return it with the root prepended rather than repeating the code.